components/RegistrationForm.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div class="container px-4 mx-auto max-w-lg">
+    <h1 class="mb-6 mt-8 font-bold text-center text-4xl">
+      Formular
+    </h1>
+    <FormulateForm>
+      <FormulateInput
+        label-class="is-required"
+        type="text"
+        label="Nume si prenume"
+        name="name"
+        validation="required"
+        :validation-messages="{
+          required: 'Campul este obligatoriu',
+        }"
+      />
+      <FormulateInput
+        label-class="is-required"
+        type="number"
+        label="Varsta"
+        name="age"
+        validation="required"
+        :validation-messages="{
+          required: 'Campul este obligatoriu',
+        }"
+      />
+      <FormulateInput
+        label-class="is-required"
+        type="number"
+        label="Telefon"
+        name="phone"
+        validation="required"
+        :validation-messages="{
+          required: 'Campul este obligatoriu',
+        }"
+      />
+      <FormulateInput
+        type="textarea"
+        label="Cum ai aflat despre intalnirile Alpha?"
+        name="info"
+      />
+      <FormulateInput
+        label-class="is-required"
+        type="checkbox"
+        wrapper-class="flex"
+        label="Am citit si sunt de acord cu politica de protectie a datelor"
+        name="gdpr"
+        validation="required"
+        :validation-messages="{
+          required: 'Campul este obligatoriu',
+        }"
+      />
+      <AlphaButton @submit="submitRegistrationForm()">
+        Trimite
+      </AlphaButton>
+    </FormulateForm>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import VueFormulate from '@braid/vue-formulate'
+import AlphaButton from './AlphaButton.vue'
+
+Vue.use(VueFormulate, {
+  classes: {
+    outer (context) {
```

Comment:
nu vrei sa adaugi cazul pentru wrapper, daca e checkbox? ca sa fie toate la fel in aplicatie, daca mai adaugam vre-odata